darwin/options.py

```diff
@@ -165,10 +165,8 @@ def __init__(self):
             "split", help="Splits a local dataset following random and stratified split types."
         )
         parser_split.add_argument("dataset", type=str, help="Local dataset name to split.")
-        parser_split.add_argument("-v", "--val-percentage", type=float, required=True, help="Validation percentage.")
-        parser_split.add_argument(
-            "-t", "--test-percentage", type=float, required=False, default=0, help="Test percentage."
-        )
+        parser_split.add_argument("val-percentage", type=float, help="Validation percentage.")
+        parser_split.add_argument("test-percentage", type=float, help="Test percentage.")
```

Comment:
Your change is changing the way the argument is passed. Let's not break it for people who use it.
```suggestion
        parser_split.add_argument("-v", "--val-percentage", type=float, help="Validation percentage.")
        parser_split.add_argument("-t", "--test-percentage", type=float, help="Test percentage.")
```

Reply:
Note that additional formatting might be needed, as the lines would get pretty long if you committed my suggestions.

Reply:
I think `required=True` will actually fix the issue. Can we try that?

Reply:
I tried that, did not work :D

Reply:
Tried myself
> $ darwin dataset split -h
usage: darwin dataset split [-h] -v VAL_PERCENTAGE -t TEST_PERCENTAGE [-s SEED] dataset

note
> -v VAL_PERCENTAGE -t TEST_PERCENTAGE
both are indicated as required now

Reply:
I don't think it's incorrect. This way `-t` and `-v` are actually NOT optional, which is what we want